### PR TITLE
Add disabled option to pipectl list application command

### PR DESCRIPTION
**Pull request by nghialv**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/app/pipectl/cmd/application/list.go [hunk 1]
```diff
@@ -52,6 +53,7 @@ func newListCommand(root *command) *cobra.Command {
 	cmd.Flags().StringVar(&c.appName, "app-name", c.appName, "The application name.")
 	cmd.Flags().StringVar(&c.envID, "env-id", c.envID, "The environment ID.")
 	cmd.Flags().StringVar(&c.appKind, "app-kind", c.appKind, fmt.Sprintf("The kind of application. (%s)", strings.Join(model.ApplicationKindStrings(), "|")))
+	cmd.Flags().BoolVar(&c.disabled, "disabled", c.disabled, "Whether to show disabled applications or not.")
```
Comment (nakabonne) [inline pkg/app/pipectl/cmd/application/list.go:56]: nit: At first it looked to return both disabled and enabled to me. I guess it's better to emphasize like:

```suggestion
	cmd.Flags().BoolVar(&c.disabled, "disabled", c.disabled, "Show only disabled applications.")
```

But it's on you.

Comment (nakabonne) [reply pkg/app/pipectl/cmd/application/list.go:56]: Hmm, this one's a little more confusing as well. Never mind!

**Comment by khanhtc1202** [comment]
LGTM
/approve

File: pkg/app/pipectl/cmd/application/list.go (hunk 1 above)
Comment (nghialv) [reply pkg/app/pipectl/cmd/application/list.go]: Nice. Just updated.

**Comment by nghialv** [comment]
@khanhtc1202 Sorry just added a minor change. Can you send another approval?

**Comment by khanhtc1202** [comment]
🙆‍♀️ 
/approve

**Pull request merged**